src/components/Header.jsx

```
@@ -1,26 +1,42 @@
 import { useCallback, useEffect, useState, memo } from "react";
 import { Link } from "react-router-dom";
 import useWindowSize from "../hooks/useWindowSize";
-import { FaCartShopping, FaBars, FaXmark, FaUser } from "react-icons/fa6";
+import { FaBagShopping, FaBars, FaXmark, FaUser } from "react-icons/fa6";
 import Nav from "./Nav";
 import Button from "./Button";
+import CartPage from "../pages/CartPage";
 
 const Header = () => {
   const [menuOpen, setMenuOpen] = useState(false);
+  const [cartOpen, setCartOpen] = useState(false);
```

ESLint

[error] 'cartOpen' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)